### ISSUE TITLE
Add @bkeepers talk

### ISSUE THREAD

**Issue by hiimbex** [probot/talks#1]
@bkeepers would you be willing to add your Chicago Constellation talk slides here?

If yes, I'm happy to add them, just let me know.

**Comment by bkeepers** [comment]
@hiimbex Yes! Here's a link to the slides, which includes a link to download the PDF in the bottom right: https://speakerdeck.com/bkeepers/automating-software-development

<blockquote><img src="https://speakerd.s3.amazonaws.com/presentations/bfd3e69baf944cbd8802b8dd8292bda1/slide_0.jpg?451313" width="48" align="right"><div><img src="https://d2dfho4r6t7asi.cloudfront.net/assets/favicon-c0fe5475f88a65b817c8cc7c0fedc01b7acc17fe27628fa13ca3f96e56e16977.png" height="14"> Speaker Deck</div><div><strong><a href="https://speakerdeck.com/bkeepers/automating-software-development">Automating Software Development</a></strong></div><div>Software is more than code, and a lot of time and overhead that goes into building software can be automated. This talk will look at the tools and techniques that software teams are using to automate everything from project management, to code review, to product feedback.

</div></blockquote>
